### PR TITLE
Restore ember 1.11.x compatibility

### PR DESCRIPTION
**Pull request by truenorth**
:tada:

### CODE REVIEW

**Comment by poteto** [comment]
Thanks @truenorth, this is awesome! I didn't know there was a polyfill for it.

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   "devDependencies": {
     "broccoli-asset-rev": "^2.0.2",
     "ember-cli": "1.13.1",
-    "ember-cli-app-version": "0.4.0",
+    "ember-cli-app-version": "0.3.5",
```
Comment (poteto) [inline package.json]: Why was this downgraded?

**Comment by truenorth** [comment]
Your co-worker wrote it :)

File: package.json (hunk 1 above)
Comment (truenorth) [inline package.json]: `ember-cli-app-version: 0.4.0` makes use of [instanceInitializers](http://emberjs.com/blog/2015/05/13/ember-1-12-released.html#toc_instance-initializers) which were introduced in `1.12.0`.

Comment (poteto) [inline package.json]: :+1:

**Pull request merged**